debs/SPECS/4.0.0/wazuh-manager/debian/postrm

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+# postrm script for Wazuh
+# Wazuh, Inc 2015-2020
+set -e
+DIR="/var/ossec"
+WAZUH_TMP_DIR="${DIR}/packages_files/manager_config_files"
+
+case "$1" in
+    remove|failed-upgrade|abort-install|abort-upgrade|disappear)
+
+    # Fix for forcing the stop of the api
+    if ps aux | grep ${DIR}/api/app.js | grep -v grep; then
+        kill -9 `ps -ef | grep "${DIR}/api/app.js" | grep -v grep | awk '{print $2}'` > /dev/null 2>&1
+    fi
+
+    if [ -f /etc/logrotate.d/wazuh-hids ]; then
+        rm -f /etc/logrotate.d/wazuh-hids
+    fi
+
+    if [ -f /etc/ossec-init.conf ]; then
+        rm -f /etc/ossec-init.conf
+    fi
+
+    if [ -d ${WAZUH_TMP_DIR} ]; then
+        rm -rf ${WAZUH_TMP_DIR}
+    fi
+
+
+
+    if [ -d /run/systemd/system ] ; then
+        systemctl disable wazuh-manager > /dev/null 2>&1
+        systemctl daemon-reload
+        systemctl reset-failed
+    fi
+
+    update-rc.d -f wazuh-manager remove
+
+    # Back up the old configuration files as .save
+    if [ ! -d ${DIR}/etc/shared/default ]; then
+        mkdir -p ${DIR}/etc/shared/default
+    fi
+
+    # Move files from ${DIR}/tmp to ${DIR}/etc
+    if [ -f ${DIR}/tmp/conffiles/shared/default/agent.conf ]; then
+        mv ${DIR}/tmp/conffiles/shared/default/agent.conf ${DIR}/etc/shared/default/agent.conf.save
+    fi
+    rm -rf ${DIR}/tmp/conffiles/shared/
+
+    # If the directory is not empty, copy the files into ${DIR}/etc
+    if [ "$(ls -A ${DIR}/tmp/conffiles)" ]; then
+        cp -Rf ${DIR}/tmp/conffiles/* ${DIR}/etc
+    fi
+    rm -rf ${DIR}/tmp/conffiles
+    rm -rf ${DIR}/tmp
+    if [ "$1" = "remove" ]; then
+        rm -rf ${DIR}/ruleset
+        rm -rf ${DIR}/var
+        rm -rf ${DIR}/logs
```

Comment:
Running `apt remove wazuh-manager` will let installed the configuration files and any other file created after the installation of the package. `rbac.db` is one of those files but, how are you going to deal with that file while removing the package? 

I mean, if you run `apt remove wazuh-manager` this db will remain installed or will be deleted?

Reply:
I have discussed this with my team and we think that db (and the SSL files) should be saved like the rest of the configuration files unless we `apt purge wazuh-manager`.